src/main/java/seedu/address/model/Model.java

```diff
@@ -67,6 +71,42 @@
      */
     MentorList getMentorList();
 
+    /* Below is the API exposed for the controllers to call */
+
+    /* Participant methods */
+
+    Participant getParticipant(Id id) throws AlfredException;
+
+    void addParticipant(Participant participant) throws Exception;
+
+    boolean updateParticipant(Id id, Participant participant);
+
+    void deleteParticipant(Id id) throws AlfredException;
```

Comment:
Sorry for not being clear, but I think delete should return the Entity deleted.

Reply:
Ah my bad, you mentioned this before, will change 👍